internal/mode/static/telemetry/collector.go

```diff
@@ -71,6 +72,15 @@ type NGFResourceCounts struct {
 	GRPCRouteCount int64
 	// BackendTLSPolicyCount is the number of relevant BackendTLSPolicies.
 	BackendTLSPolicyCount int64
+	// GatewayAttachedClientSettingsPolicyCount is the number of relevant ClientSettingsPolicies
+	// attached at the Gateway level.
+	GatewayAttachedClientSettingsPolicyCount int64
+	// RouteAttachedClientSettingsPolicyCount is the number of relevant ClientSettingsPolicies attached at the Route level.
+	RouteAttachedClientSettingsPolicyCount int64
+	// ObservabilityPolicyCount is the number of relevant ObservabilityPolicies.
+	ObservabilityPolicyCount int64
+	// NGINXProxyCount is the number of NGINXProxies.
+	NGINXProxyCount int64
```

Comment:
For consistency with the code base, let's use "Nginx" not "NGINX" in variable/field names.